aderyn_core/src/context/loader.rs

```diff
@@ -58,6 +58,618 @@ pub enum ASTNode {
     WhileStatement(WhileStatement),
 }
 
+impl From<ArrayTypeName> for ASTNode {
```

Comment:
I merged the previous PR #122, and the diff here is similar. Could you rebase this branch based on the latest `dev` branch so these repeated changes aren't present?

Reply:
Done Alex! 👊